Artifacts/MsSql/Data/Security/1060-RemoveRedundantBootstrapPermissions.sql

```diff
@@ -0,0 +1,64 @@
+    -- SPDX-License-Identifier: Apache-2.0
+    -- Licensed to the Ed-Fi Alliance under one or more agreements.
+    -- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+    -- See the LICENSE and NOTICES files in the project root for more information.
+
+    DECLARE @claim_Name NVARCHAR(MAX) = 'http://ed-fi.org/ods/identity/claims/domains/educationOrganizations';
+    DECLARE @claimSet_Name NVARCHAR(MAX) = 'Bootstrap Descriptors and EdOrgs';
+    DECLARE @educationOrganizationsResourceClaimId INT;
+    DECLARE @claimSet_Id INT;
+    DECLARE @msg NVARCHAR(MAX);
+    DECLARE @messageText NVARCHAR(MAX);
+    DECLARE @createActionId INT;
+    DECLARE @authorizationStrategyId INT;
+
+    SELECT @authorizationStrategyId=AuthorizationStrategyId FROM dbo.AuthorizationStrategies where DisplayName='No Further Authorization Required'
+    SELECT @createActionId =ActionId FROM dbo.Actions where ActionName='Create'
+
+    --New message for claimname does not exist error
+    EXEC sys.sp_addmessage
+    @msgnum = 50001,
+    @severity = 1,
+    @msgtext = N'ClaimName ''%s'' does not exist.',
+    @with_log = true,
+    @replace = 'REPLACE',
+    @lang = 'us_english';
+
+    IF NOT  EXISTS (SELECT 1 FROM dbo.ResourceClaims WHERE ClaimName = @claim_Name)
+    BEGIN
+        SET @messageText =  FORMATMESSAGE(50001,@claim_Name);
+        THROW 50001, @messageText, 1
+    END
+
+    --New message for ClaimSetName does not exist error
+    EXEC sys.sp_addmessage
```

Comment:
Again, don't add custom messages. Use formatted string variables.

Reply:
I fixed it . Thanks